app/workers/deletes.php

```diff
@@ -321,10 +321,10 @@ protected function deleteExpiredSessions(): void
     {
         $consoleDB = $this->getConsoleDB();
 
-        $this->deleteForProjectIds(function (string $projectId) use ($consoleDB) {
-            $dbForProject = $this->getProjectDB($projectId);
+        $this->deleteForProjectIds(function (Document $project) use ($consoleDB) {
```

Comment:
When I opened code locally it gives error:

<img width="1068" alt="CleanShot 2022-12-14 at 11 38 18@2x" src="https://user-images.githubusercontent.com/19310830/207573147-113e4aa1-4526-4ae8-afe4-1b768719bd9c.png">


Maybe this is `db-pools` specific problem only.